.changeset/wicked-doors-grab.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": patch
+"@khanacademy/perseus-editor": patch
+---
+
+Remove all references to multipleNumberInput (ie. Ask for (one|all) correct solution)
```

Comment:
I've chosen `patch` because the underlying implementation is already gone... so this option controlled nothing even though it was still surfaced in the editor.